bin/generate-npm-tag.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+set -e
+
+# Get highest tag published on Github
+HIGHEST_PUBLISHED_VERSION=$(git tag --list 2>/dev/null | sort -V | tail -n1 2>/dev/null | sed 's/v//g')
+
+# Extract tag version from ./package/package.json
+CURRENT_TAG=$(node -p "require('./package/package.json').version")
```

Comment:
As above comment about HIGHEST_PUBLISHED_TAG

```suggestion
CURRENT_VERSION=$(node -p "require('./package/package.json').version")
```